cli/scorecard/violations.go

```diff
@@ -135,6 +138,74 @@ func getViolations(inventory *InventoryConfig, config *ScoringConfig) ([]*RichVi
 	return richViolations, nil
 }
 
+// getViolationsConcurrently finds all Config Validator violations for a given Inventory concurrently
```

Comment:
I'm not sure if we should introduce a new function as it introduces two different code paths which need to be kept in sync. I suggest we simply use the same function for both (but by default only have a single worker).